src/rgw/rgw_rados.cc

```diff
@@ -4412,6 +4412,12 @@ int RGWRados::copy_obj(RGWObjectCtx& obj_ctx,
   if (lh != attrs.end())
     src_attrs[RGW_ATTR_OBJECT_LEGAL_HOLD] = lh->second;
 
+  if (dest_bucket_info.flags & BUCKET_VERSIONS_SUSPENDED) {
+    src_attrs.erase(RGW_ATTR_OLH_ID_TAG);
+    src_attrs.erase(RGW_ATTR_OLH_INFO);
+    src_attrs.erase(RGW_ATTR_OLH_VER);
+  }
```

Comment:
FYI `dest_bucket_info` doesn't seem to be defined here, maybe review the other backports as well (thanks to Alexander Patrakov for pointing it out to me)

Reply:
Fixed. 
`make check` failed though. But it looks not related the changes in this PR.